src/main/java/org/jenkinsci/plugins/sonargerrit/inspection/entity/Report.java

```diff
@@ -44,6 +44,26 @@ public String getVersion() {
         return users;
     }
 
+    public void setVersion(String version) {
+        this.version = version;
+    }
+
+    public void setIssues(List<Issue> issues) {
```

Comment:
For Issue.creationDate defensive copying is used, should this also be done here?

Reply:
Interesting, but didn't find any location in code where issues are modified apart from `issue.setNew(true)`